DataBox Api version upgrade to 2022-02-01

sashanm opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [ ] SDK of Js
      - [x] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

sashanm committed:
first commite

ruowan commented:
/azp run

vaishnavi-janardhan committed:
Fix for spellcheck

vaishnavi-janardhan committed:
Fix for spell check and revert edits

vaishnavi-janardhan commented:
/azp run

vaishnavi-janardhan commented:
Hi @ruowan, none of the failing checks seem to be arising due to the files being added in this PR. Could you please review the failing checks?

vaishnavi-janardhan commented:
Dear review board, please find below certain observations on the CI failure
- [Swagger SDK breaking change tracking](https://github.com/Azure/azure-rest-api-specs/pull/18285/checks?check_run_id=5814616505) - This failure doesn't seem to be related to the changes in the current PR. We have been seeing the same failure in our previous PRs as well ([PR upgrade to 2021-12-01](https://github.com/Azure/azure-rest-api-specs/pull/16989))
- [SDK azure-sdk-for-net] - 2 failures in this check were seen in the previous PR as well. The failures are with the test code in the azure-sdk-for-net repo. Should we modify the test code before the swagger PR gets checked in.

Could you please look into this and advise us on how to fix these failures. Thank you!

mentat9 commented:
@sashanm - Please either create your new API version PR using OpenAPIHub or manually create it using the PR format for new API versions:
1. Copy all files from current API version to new API version folder.
2. Create first commit.
3. Apply changes for new API version in subsequent commits including updating API version in files.

vaishnavi-janardhan commented:
@mentat9 I've created [a new PR](https://github.com/Azure/azure-rest-api-specs/pull/18734) following the steps that you have mentioned above. Could you please review it?

BigCat20196 committed:
Update readme.python.md

`readme.python` has used track2 by default, so it no longer needs `track2` tag.

mentat9 commented:
> @mentat9 I've created [a new PR](https://github.com/Azure/azure-rest-api-specs/pull/18734) following the steps that you have mentioned above. Could you please review it?

Yes, I've added the new one to my queue. Please close this one.
